### PR TITLE
Add tag example in the search bar

### PR DESCRIPTION
**Pull request by sueann**
## What changes are proposed in this pull request?

To make it more obvious that you can search using tags, added an example of it in the search bar's placeholder text.

## How is this patch tested?

Dev UI

### Is this a user-facing change?

- [ ] No. You can skip the rest of this section.
- [X] Yes. Give a description of this change to be included in the release notes for MLflow users.

### What component(s) does this PR affect?

- [X] UI
- [ ] CLI
- [ ] API
- [ ] REST-API
- [ ] Examples
- [ ] Docs
- [ ] Tracking
- [ ] Projects
- [ ] Artifacts
- [ ] Models
- [ ] Scoring
- [ ] Serving
- [ ] R
- [ ] Java
- [ ] Python

### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [X] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

**Comment by sueann** [comment]
Screen Shot 2019-11-07 at 10 22 12 PM

File: mlflow/server/js/src/components/ExperimentView.js [hunk 1]
```diff
@@ -413,7 +413,8 @@ export class ExperimentView extends Component {
                     <input
                       className="ExperimentView-searchInput"
                       type="text"
-                      placeholder={'metrics.rmse < 1 and params.model = "tree"'}
+                      placeholder={'metrics.rmse < 1 and params.model = "tree" and ' +
+                                   'tags."mlflow.source.name" = "local"'}
```
Comment (mparkhe) [inline mlflow/server/js/src/components/ExperimentView.js]: Did you intend for this to be "mlflow.source.type" = "LOCAL"?

[System Tags](https://mlflow.org/docs/latest/tracking.html#system-tags) in MLflow docs

**Comment by sueann** [comment]
Screen Shot 2019-11-07 at 11 59 18 PM

**Review by mparkhe — APPROVED** [review]
LGTM!

**Pull request merged**